src/cli.ts

```diff
@@ -151,20 +152,16 @@ program
     }
   });
 
+const apiDocsLink =
+  'API key used for Kibana authentication(https://www.elastic.co/guide/en/kibana/master/api-keys.html).';
+
 // Push command
 program
   .command('push')
   .description(
     'Push all journeys in the current directory to create monitors within the Kibana monitor management UI'
   )
-  .addOption(
-    new Option(
-      '--auth <auth>',
-      'API key used for Kibana authentication(https://www.elastic.co/guide/en/kibana/master/api-keys.html).'
-    )
-      .env('SYNTHETICS_API_KEY')
-      .makeOptionMandatory(true)
-  )
+  .addOption(new Option('--auth <auth>', apiDocsLink).makeOptionMandatory(true))
```

Comment:
can you move this option to the shared list here - https://github.com/elastic/synthetics/blob/c560bef082718c9799b93da72a3af95bba4c225f/src/options.ts#L171

Reply:
actually we can't reuse it, since once it's made mandatory, it will becomes mandatory for auth as well

Reply:
But we can enable/disable auth when adding. Pushed the change